src/content/docs/it/tutorial/2-pages/1.mdx

```diff
@@ -0,0 +1,162 @@
+  Tutorial: Crea il tuo primo blog Astro —
+  Aggiungi nuove pagine al tuo sito con link di navigazione tra di esse
+i18nReady: true
+---
+import Checklist from '~/components/Checklist.astro';
+import Blanks from '~/components/tutorial/Blanks.astro';
+import Box from '~/components/tutorial/Box.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+Ora che sai che i file `.astro` sono responsabili delle pagine sul tuo sito web, è il momento di crearne uno!
+
+<PreCheck>
+  - Crea due nuove pagine sul tuo sito web: About e Blog
+  - Aggiungi link di navigazione alle tue pagine
+  - Distribuisci una versione aggiornata del tuo sito web sul web
+</PreCheck>
+
+## Crea un nuovo file `.astro`
+
+<Steps>
+1. Nel pannello dei file del tuo editor di codice, naviga fino alla cartella `src/pages/` dove vedrai il file esistente `index.astro`
+
+2. Nella stessa cartella, crea un nuovo file chiamato `about.astro`.
+
+3. Copia o ridigita il contenuto di `index.astro` nel tuo nuovo file `about.astro`.
+
+    :::tip
+    Il tuo editor potrebbe mostrare un cerchio bianco pieno sull'etichetta della scheda per questo file. Questo significa che il file non è ancora stato salvato. Nel menu File in VS Code, abilita "Salvataggio automatico" e non dovresti più aver bisogno di salvare manualmente alcun file.
+    :::
+
+4. Aggiungi `/about` alla fine dell'URL di anteprima del tuo sito web nella barra degli indirizzi e verifica di poter vedere una pagina caricarsi lì. (es. `http://localhost:4321/about`)
+</Steps>
+
+In questo momento, la tua pagina "About" dovrebbe apparire esattamente come la prima pagina, ma stiamo per cambiarlo!
+
+## Modifica la tua pagina
+
+Modifica il contenuto HTML per rendere questa pagina su di te.
+
+Per cambiare o aggiungere più contenuto alla tua pagina About, aggiungi più tag di elementi HTML contenenti contenuto. Puoi copiare e incollare il codice HTML qui sotto tra i tag `<body></body>` esistenti, o creare il tuo.
+
+```astro title="src/pages/about.astro" ins={3-8} del={2}
+<body>
+  <h1>Il mio sito Astro</h1>
+  <h1>Su di me</h1>
+  <h2>... e il mio nuovo sito Astro!</h2>
+
+  <p>Sto seguendo il tutorial introduttivo di Astro. Questa è la seconda pagina sul mio sito web, ed è la prima che ho costruito da solo!</p>
+
+  <p>Questo sito si aggiornerà man mano che completerò altre parti del tutorial, quindi continua a controllare e vedi come sta andando il mio percorso!</p>
+</body>
+```
+
+Ora, visita di nuovo la tua pagina `/about` nella scheda del tuo browser, e dovresti vedere il tuo contenuto aggiornato.
+
+## Aggiungi link di navigazione
+
+Per rendere più facile visualizzare in anteprima tutte le tue pagine, aggiungi link di navigazione di pagina HTML prima del tuo `<h1>` nella parte superiore di entrambe le tue pagine (`index.astro` e `about.astro`):
+
+```astro title="src/pages/about.astro" ins={1-2}
+<a href="/">Home</a>
+<a href="/about/">About</a>
```

Comment:
Maybe this could also be translated like the rest of the text in this code block? Or maybe as a native speaker you think it makes more sense to keep it as-is?